docs/source/examples/checks/vision/performance/performance_report.ipynb

```diff
@@ -5,48 +5,52 @@
    "id": "e767e792",
```

Comment:
<p>alreaady -&gt; already </p><p>Try out the spell checker extension(:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>